components/applications-service/pkg/storage/postgres/services.go

```diff
@@ -242,15 +248,18 @@ func (db *Postgres) GetServicesDistinctValues(fieldName, queryFragment string) (
 	}
 
 	columnName := columnNameForField(fieldName)
-	query := fmt.Sprintf("SELECT DISTINCT %s from service_full AS t WHERE t.%s ILIKE $1 ORDER BY %s ASC LIMIT 100;",
+	queryFirst := fmt.Sprintf("SELECT DISTINCT %s from service_full AS t WHERE t.%s ILIKE $1 ",
 		columnName,
 		columnName,
+	)
+	querySecond := fmt.Sprintf(" ORDER BY %s ASC LIMIT 100;",
 		columnName,
 	)
+	query := queryFirst + whereConstraints + querySecond
 	matcher := fmt.Sprintf("%s%%", queryFragment)
```

Comment:
Do we need to do any verification of this field? or it doesn't matter what the front end sends to the backend?